android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -804,6 +804,16 @@ class XMTPModule : Module() {
             }
         }
 
+        AsyncFunction("addedByAddress") Coroutine { clientAddress: String, id: String ->
+            withContext(Dispatchers.IO) {
+                logV("addedByAddress")
+                val client = clients[clientAddress] ?: throw XMTPException("No client")
+                val group = findGroup(clientAddress, id)
+
+                group?.addedByAddress()
```

Comment:
We should throw if it can't find the group instead of returning null. Just like you did in iOS

Reply:
Syntax wise you could probably do
```suggestion
                group?.addedByAddress() ?: throw XMTPException("No group found")
```